core/templates/dev/head/filters.js

```diff
@@ -441,3 +441,15 @@ oppia.filter('removeDuplicatesInArray', [function() {
     });
   };
 }]);
+
+oppia.filter('stripFormatting', [function() {
+  return function(html, whitelistedImgClasses) {
+    // Strip out anything between and including <>,
+    // unless it is an img whose class includes one of the whitelisted classes.
+    var regex = new RegExp(
+      '(?!<img.*class=".*(' + whitelistedImgClasses.join('|') +
```

Comment:
- Add a space char after 'img' (to avoid matching tags like imga)
- Add a space char before 'class'